runners/flink/src/test/java/org/apache/beam/runners/flink/streaming/ExecutableStageDoFnOperatorTest.java

```diff
@@ -225,18 +229,22 @@ public String getId() {
               @Override
               public Map<String, FnDataReceiver<WindowedValue<?>>> getInputReceivers() {
                 return ImmutableMap.of(
-                    "pCollectionId",
+                    "input",
                     input -> {
                       /* Ignore input*/
                     });
               }
 
               @Override
               public void close() throws Exception {
+                if (onceEmitted) {
```

Comment:
Why is this required? Duplicate call to `close()`?

Reply:
Close is only called once per bundle but there are two bundles now due to the watermark processing.